docs/user/helper.rst

```diff
@@ -19,6 +19,35 @@ length of the arrays and not the number of keys, and you get a
 with :class:`.SliceDict`, this works.
 
 
+.. _slicedictx:
```

Comment:
The class name is `SliceDatasetX`, but the anchor name has _dict_ in it. Is this on purpose?

Reply:
Thanks, this was unintended